src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -141,8 +141,6 @@ public enum OperationKind
         AddressOfExpression = 0x203,
         /// <summary>Indicates an <see cref="IPointerIndirectionReferenceExpression"/>.</summary>
         PointerIndirectionReferenceExpression = 0x204,
-        /// <summary>Indicates an <see cref="IUnboundLambdaExpression"/>.</summary>
-        UnboundLambdaExpression = 0x205,
```

Comment:
If there is a concern that that could be a breaking change, we could fill the gap with a member that wasn't shipped yet, or add a clear comment that the number is available for reuse in the future.